cfme/tests/containers/test_containers_data_integrity_properties.py

```diff
@@ -0,0 +1,114 @@
+
+pytestmark = [
+    pytest.mark.uncollectif(
+        lambda: current_version() < "5.6"),
+    pytest.mark.usefixtures('setup_provider'),
+    pytest.mark.tier(1)]
+pytest_generate_tests = testgen.generate(
+    testgen.container_providers, scope='function')
+
+CONTAINERS_IMAGES_FIELDS = ['Name', 'Image Id', 'Full Name']
+CONTAINERS_NODES_FIELDS = ['Name', 'Creation timestamp', 'Resource version', 'Number of CPU Cores',
+                           'Memory', 'Max Pods Capacity', 'System BIOS UUID', 'Machine ID',
+                           'Infrastructure Machine ID', 'Runtime version', 'Kubelet version',
+                           'Proxy version', 'Operating System Distribution', 'Kernel version']
+CONTAINERS_FIELDS = ['Name', 'State', 'Restart count', 'Backing Ref (Container ID)', 'Privileged']
+IMAGE_REGISTRY_FIELDS = ['Host']
+
+
+# CMP-9945
+
+@pytest.mark.parametrize('container_fields', product(CONTAINERS_FIELDS))
+def test_containers_data_containers_integrity_properties(container_fields):
+    """ Container details page > Properties fields
+        This test checks correct population of the Properties Fields in Containers' details menu
+        Steps:
+            * Goes to Containers -- > Containers menu
+            * Go through each Container in the menu and check validity of Properties fields
+        """
+    sel.force_navigate('containers_containers')
+    container_name = [r.name.text for r in list_tbl.rows()]
+    container_pod_name = [r.pod_name.text for r in list_tbl.rows()]
+    dictionary = dict(zip(container_name, container_pod_name))
+    for container_name, container_pod_name in dictionary.iteritems():
+        pod = Pod(container_pod_name, Provider)
+        cont = Container(container_name, pod)
+        val = cont.get_detail('Properties', ''.join(container_fields))
+        assert val
+        sel.click("//i[@class='fa fa-arrow-left fa-lg']")
+        sel.click("//a[@data-click='history_choice__history_1']")
+
+
+# CMP-9988
+# Check how to add an Image Registry with code
+@pytest.mark.parametrize('image_registry_fields', product(IMAGE_REGISTRY_FIELDS))
+def test_containers_image_registries_integrity_properties(provider, image_registry_fields):
+    """ Image Registry details page > Properties fields
+        This test checks correct population of the Properties Fields in Image Registry's
+        details menu.
+        Prerequisites: Image Registries in CFME
+        Steps:
+            * Goes to Containers -- > Image Registries menu
+            * Go through each Image Registry in the menu and check validity of Properties fields
+        """
+    list_tbl = CheckboxTable(table_locator="//div[@id='list_grid']//table")
+    sel.force_navigate('containers_image_registries')
+    image_registry_hosts = [r.host.text for r in list_tbl.rows()]
+    for host in image_registry_hosts:
+        obj = ImageRegistry(host, provider)
+        val = obj.get_detail('Properties', ''.join(image_registry_fields))
+        assert val
```

Comment:
At the moment I was asked just to assert that there is any value but not to assert that the value is correct and is verified against something from the DB or anywhere else